.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+This project is for Ruby version 2.3 and older.
```

Comment:
I'll change this. However the markdown syntax is interpreted not before the issue is posted.